src/utils/helpers/list/buildQueryFilters.ts

```diff
@@ -139,6 +144,9 @@ const mapQueryToFilterField: FilterMap = {
 
 const splitCommaList = (values: string) => values.split(',')
 
+// Use this to find string in an array of strings
+const inArray = (value: string) => ({ anyEqualTo: value })
```

Comment:
I've checked another filter type, which can accept an array:

```GraphQL
query MyQuery2 {
  applicationList(
    filter: {
      reviewerUsernames: {
        overlaps: ["testReviewer2", "testReviewer1"]
      }
    }
  ) {
    nodes {
      id
      reviewerUsernames
    }
  }
}

```

this seems to work correctly, would be great to use this one (split values into array, similar to enum list filtering)

Reply:
I was looking for it! Thanks, I'll add this implementation to this PR now : )

Reply:
Done: 8fe1bd8